bins/revme/src/cmd/statetest/models/mod.rs

```diff
@@ -128,15 +109,15 @@ pub struct TransactionParts {
     #[serde(default)]
     pub access_lists: Vec<Option<AccessList>>,
     #[serde(default)]
-    pub authorization_list: Vec<Authorization>,
+    pub authorization_list: Vec<TestAuthorization>,
```

Comment:
Can you try if parsing passes with removal of `#[serde(default)]` and with field looking like `Option<Vec<TestAuthorization>>`

Reply:
Okay I made it optional and this is what the runner.rs snippet code is like, 
![image](https://github.com/user-attachments/assets/f189c02b-5efb-47dc-8321-7b70f13ba31f)
it still has the `continue` statement because the original code does in the main branch, Let me know if you wanted something else.

Reply:
It should be something like `env.tx.authorization_list = auth.map(|list| list.iter().map(|auth| auth.into_recovered()).collect());`